### PR TITLE
fix 462

### PR DESCRIPTION
**Pull request by kwk**
- **Disable building standalone builds. See #462**
- **Disable checking standalone builds. See #462**
- **Fork big-merge copr builds into @fedora-llvm-team/llvm-snapshots. See #462**
- **Get rid of llvm-snapshot-builder. See #462**
- **Update copr project documentation about new name for incubator projects. See #462**
- **Make big-merge the default strategy in config. See #462**

### CODE REVIEW

File: .github/workflows/fedora-copr-build.yml [hunk 1]
```diff
@@ -13,28 +13,25 @@ jobs:
...
+          #   copr_project_tpl: "@fedora-llvm-team/llvm-snapshots-incubator-YYYYMMDD"
+          #   copr_target_project: "@fedora-llvm-team/llvm-snapshots"
+          #   clone_url_tpl: "https://src.fedoraproject.org/rpms/PKG.git"
+          #   clone_ref: upstream-snapshot
           - name: big-merge
             copr_project_tpl: "@fedora-llvm-team/llvm-snapshots-big-merge-YYYYMMDD"
-            copr_target_project: "@fedora-llvm-team/llvm-snapshots-big-merge"
+            copr_target_project: "@fedora-llvm-team/llvm-snapshots"
             extra_script_file: "scripts/functions-big-merge.sh"
             clone_url_tpl: "https://src.fedoraproject.org/forks/kkleine/rpms/PKG.git"
```
Comment (nikic) [inline .github/workflows/fedora-copr-build.yml:27]:
```suggestion
            clone_url_tpl: "https://src.fedoraproject.org/rpms/PKG.git"
```
I think this should point to the main repo now?

Comment (kwk) [reply .github/workflows/fedora-copr-build.yml:27]: That is correct. Let me fix it.

**Review by nikic — APPROVED** [review]
LGTM if we're okay with temporarily publishing to llvm-snapshots without lld.

If not, then I think we first need to only do the change of the repo, then add lld support, make sure that it builds, and only then start promoting from the big-merge branch. Or alternatively build lld separately in the big-merge incubator.

**Comment by nikic** [comment]
I've put up https://github.com/fedora-llvm-team/llvm-snapshots/pull/473 with just the repo change. I'd like to get the big-merge branch back to working ASAP without blocking it on the lld work.

**Pull request merged**